lib/apm.js

```diff
@@ -1,5 +1,13 @@
 'use strict';
 const EventEmitter = require('events').EventEmitter;
+const fs = require('fs');
+const path = require('path');
+const Readable = require('stream').Readable;
+
+function stringify(logObj) {
+  const EJSON = require('mongodb-extjson');
```

Comment:
ah! I remember now why I was waiting on this PR. We're merging extjson into bson for the v4 release, so I was holding off until then to clarify some of this "optional dependency on extjson" business

Reply:
/cc @daprahamian ^^

Reply:
We're still on BSON v 1 in the driver. Unless we're upgrading to BSON v4 in the next release (which I am hesitant to do), this should be fine.